test/powershell/Modules/Microsoft.PowerShell.Utility/Export-Alias.Tests.ps1

```diff
@@ -120,39 +135,39 @@ Describe "Export-Alias DRT Unit Tests" -Tags "CI" {
 		{
 			chmod 777 $fulltestpath
 		}
-		
+
     }
 }
 
 Describe "Export-Alias" -Tags "CI" {
-    $testAliasDirectory = Join-Path -Path $TestDrive -ChildPath ExportAliasTestDirectory
-    $testAliases        = "TestAliases"
-    $fulltestpath       = Join-Path -Path $testAliasDirectory -ChildPath $testAliases
+
+	BeforeAll {
+		$testAliasDirectory = Join-Path -Path $TestDrive -ChildPath ExportAliasTestDirectory
+		$testAliases        = "TestAliases"
+		$fulltestpath       = Join-Path -Path $testAliasDirectory -ChildPath $testAliases
+	}
 
     BeforeEach {
-	New-Item -Path $testAliasDirectory -ItemType Directory -Force
+		New-Item -Path $testAliasDirectory -ItemType Directory -Force
     }
 
     It "Should be able to create a file in the specified location"{
-	Export-Alias $fulltestpath
-
-	Test-Path $fulltestpath | Should be $true
+		Export-Alias $fulltestpath
+		Test-Path $fulltestpath | Should be $true
     }
 
     It "Should create a file with the list of aliases that match the expected list" {
-	Export-Alias $fulltestpath
+		Export-Alias $fulltestpath
+		Test-Path $fulltestpath | Should Be $true
 
-	Test-Path $fulltestpath | Should Be $true
-
-	$actual   = Get-Content $fulltestpath | Sort-Object
-	$expected = Get-Command -CommandType Alias
-	
-	for ( $i=0; $i -lt $expected.Length; $i++)
-	{
-	    # We loop through the expected list and not the other because the output writes some comments to the file.
-	    $expected[$i] | Should Match $actual[$i].Name
-	}
+		$actual   = Get-Content $fulltestpath | Sort-Object
+		$expected = Get-Command -CommandType Alias
 
+		for ( $i=0; $i -lt $expected.Length; $i++)
+		{
+			# We loop through the expected list and not the other because the output writes some comments to the file.
+			$expected[$i] | Should Match $actual[$i].Name
+		}
     }
 
     Remove-Item -Path $testAliasDirectory -Recurse -Force

```

Comment:
>Remove-Item -Path $testAliasDirectory -Recurse -Force [](start = 4, length = 53)

this should probably be removed as it's done automatically when `$TESTDRIVE` is cleaned up